tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -387,6 +387,46 @@ function test_full_sync_sends_all_post_meta() {
 		$this->assertEquals( array( 'foo', 'bar' ), $this->server_replica_storage->get_metadata( 'post', $post_id, 'test_meta_array', true ) );
 	}
 
+	function test_full_sync_doesnt_sends_forbiden_private_or_public_post_meta() {
+		$post_id = $this->factory->post->create();
+		// forbidden private meta
+		add_post_meta( $post_id, '_test_meta_key', 'foo1' );
+		add_post_meta( $post_id, '_test_meta_array', array( 'foo2', 'bar' ) );
```

Comment:
see below, we test syncing blacklisted "snapTW" and whitelisted "_wp_attachment_metadata"